docs/content/docs/grype/_index.md

```diff
@@ -1,49 +1,47 @@
 ---
-title: "Beta - Grype Integration"
+title: "Grype Integration"
 linkTitle: "Grype Vulnerability Scanner"
 weight: 1
 ---
 
-Anchore Engine 0.10 includes a limited-functionality beta release of an integration with [Grype](https://github.com/anchore/grype)
-for vulnerability scanning. This integration will replace the legacy vulnerability scanner in a future version of Anchore Engine,
-but is provided in this release in a preview capacity so users can try it out.
+As of Anchore Engine 1.0.0, Anchore Engine is fully integrated with Grype by default for vulnerability scanning. The V2 vulnerability scanner, based on Grype, replaces the legacy vulnerability scanner in previous versions of Anchore Engine.
+You can keep the legacy vulnerability scanner when installing Anchore Engine 1.0.0, but you will have to explicitly configure 1.0.0 to use the legacy vulnerability scanner.
```

Comment:
"you will have to explicitly configure 1.0.0 to use the legacy vulnerability scanner" is this correct? I thought the plan (this may have changed over time) was that existing users could upgrade to 1.0.0 and keep the legacy scanner without having to change their config. I.e. upgrading and changing to the grype provider is what requires updating your helm chart.

Reply:
See this thread in slack:  https://anchore.slack.com/archives/C02D6U7KS4W/p1632356329335500 for more context.

Reply:
Hi @dspalmer99. I think this is confusing. From my understanding from @DaneWeber, it differs depending between a new install and an existing customer upgrading.
A new install comes with Grype set as the default. If a customer wants to use the legacy feed/scanner, then they will have to configure the install to use legacy.
For an existing custoer who is upgrading, they will retain their previous vulnerability scanner settings (legacy). They can also follow linked instructions to upgrade to the new Grype vulnerability scanner. 
At least this is my understanding.